.github/workflows/ci.yaml

```diff
@@ -253,17 +257,96 @@ jobs:
             --build-arg DISTRO_NAME \
             --build-arg DISTRO_VER \
             -t condaforge/$DOCKERIMAGE:$DOCKERTAG \
+            -t ${{ env.REGISTRY_DH }}/$DOCKERIMAGE:$DOCKERTAG \
+            -t ${{ env.REGISTRY_QUAY }}/$DOCKERIMAGE:$DOCKERTAG \
             -f ${DOCKERFILE:-${DOCKERIMAGE}}/Dockerfile \
             --no-cache --squash .
 
       - name: Run image
         run: |
           ./.circleci/run_docker_build.sh
 
-      - name: Deploy
+      - name: Deploy images
         if: github.ref == 'refs/heads/main' && github.repository == 'conda-forge/docker-images'
         env:
+          CFD_QUAY_USER: conda_forge_daemon
           CFD_QUAY_PASSWORD: ${{ secrets.CFD_QUAY_PASSWORD }}
+          DH_USER: condaforgebot
           DH_PASSWORD: ${{ secrets.DH_PASSWORD }}
         run: |
           ./scripts/deploy
+  
+  build-manifest:
+    needs: [build]
+    if: github.ref == 'refs/heads/main' && github.repository == 'conda-forge/docker-images'
+    name: ${{ matrix.docker-registry }}/${{ matrix.cfg.DOCKER_MANIFEST }}:${{ matrix.cfg.DOCKER_TAG }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        docker-registry: [docker.io/condaforge, quay.io/condaforge]
+        cfg:
+          - DOCKER_MANIFEST: linux-anvil-cos7
+            DOCKER_TAG: "latest"
+            DOCKER_IMAGES: "<<ORG>>/linux-anvil-cos7-x86_64:<<TAG>>,<<ORG>>/linux-anvil-ppc64le:<<TAG>>,<<ORG>>/linux-anvil-aarch64:<<TAG>>"
+          - DOCKER_MANIFEST: linux-anvil-alma
+            DOCKER_TAG: "8"
+            DOCKER_IMAGES: "<<ORG>>/linux-anvil-alma-x86_64:<<TAG>>,<<ORG>>/linux-anvil-alma-ppc64le:<<TAG>>,<<ORG>>/linux-anvil-alma-aarch64:<<TAG>>"
+          - DOCKER_MANIFEST: linux-anvil-ubi-cuda
+            DOCKER_TAG: "11.0"
+            DOCKER_IMAGES: "<<ORG>>/linux-anvil-cuda:<<TAG>>,<<ORG>>/linux-anvil-ppc64le-cuda:<<TAG>>,<<ORG>>/linux-anvil-aarch64-cuda:<<TAG>>"
+          - DOCKER_MANIFEST: linux-anvil-cos7-cuda
```

Comment:
There is some repetition for all `DOCKER_MANIFEST: linux-anvil-cos7-cuda` but I wasn't able to compose a more compact matrix (e.g. by using `DOCKER_TAG: ["11.1", "11.2", "11.3", ...]`).

Reply:
That's ok. We already have some repetition for the CUDA images

Also longer term we are slowly phasing the CUDA images out entirely. It will take us some time to get there though

We could probably take this opportunity to drop CUDA 11.0 & 11.1 as we don't use those any more ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1708 )

Reply:
Do you suggest to drop them in this PR or in another ?

Reply:
We don't need to do in this PR. Just if they are causing difficulty, we should feel comfortable dropping them